config/routes.rb

```diff
@@ -1,5 +1,4 @@
 Rails.application.routes.draw do
   mount DDS::Base, at: '/'
   get "apiexplorer" => "swaggerui#index"
-  root to: redirect("/portal", status: 302)
```

Comment:
@dmlond where do you think the root url should redirect to? The apidocs or apiexplorer? I think I'm leaning more towards the former, since it contains more information about DDS.